<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task1Window</class>
 <widget class="QDialog" name="Task1Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите X, Y, Z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Результат:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCalculate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Получить результат</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
